Don't install KDE config files if configure didn't find a place for them

Submitted upstream at:
https://sourceforge.net/tracker/?func=detail&aid=2767142&group_id=67131&atid=517001

diff -urp -x '*~' -x '*.orig' reallyslick/cpp_src/Makefile.am reallyslick/cpp_src/Makefile.am
--- reallyslick/cpp_src/Makefile.am	2008-08-31 18:22:00.000000000 -0700
+++ reallyslick/cpp_src/Makefile.am	2009-04-15 16:27:45.972615000 -0700
@@ -20,7 +20,7 @@ if INSTALL_XSCREENSAVER_CONFIG_FILES
 dist_config_DATA = cyclone.xml fieldlines.xml flocks.xml flux.xml helios.xml hyperspace.xml plasma.xml skyrocket.xml solarwinds.xml
 endif
 
-if INSTALL_XSCREENSAVER_CONFIG_FILES
+if INSTALL_KDE_CONFIG_FILES
 dist_kdessconfig_DATA = cyclone.desktop fieldlines.desktop flocks.desktop flux.desktop helios.desktop hyperspace.desktop plasma.desktop skyrocket.desktop solarwinds.desktop
 endif
 
